Sources/LocalNetworkConnectivity/VLCNetworkServerBrowserViewController.m

```diff
@@ -85,7 +85,8 @@ - (void)networkServerBrowser:(id<VLCNetworkServerBrowser>)networkBrowser request
                                          errorMessage:NSLocalizedString(@"LOCAL_SERVER_CONNECTION_FAILED_MESSAGE", nil)
                                        viewController:self
                                         buttonsAction:@[[[VLCAlertButton alloc] initWithTitle: NSLocalizedString(@"BUTTON_OK", nil)
-                                               action: ^(UIAlertAction* action){}]]];
+                                                                                        style: UIAlertActionStyleDefault
```

Comment:
you shouldn't need that with the default param right?

Reply:
Sadly, I believe it's not possible in Objective-c. 😢